tests/unit_tests/test_wave_06.py

```diff
@@ -109,7 +118,19 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
+    assert result == True
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_c in jesse.inventory
```

Comment:
You can combine lines 123 and 124 if you want:
`assert len(tai.inventory) == 3 and len(jesse.inventory) == 3`

You can use list destructing and write lines 125-130 lines like this too - a pythonic way of doing it:
```python
assert [item_f, item_a, item_b] == tai.inventory
assert [item_d, item_e, item_c] == jesse.inventory
```

https://medium.com/@umaramanat66/destructuring-list-in-python-like-javascript-f7d4c0968538